src/helpers/userHelper.js

```diff
@@ -1,59 +1,60 @@
-const mongoose = require('mongoose');
-const moment = require('moment-timezone');
-const _ = require('lodash');
-const userProfile = require('../models/userProfile');
-const timeEntries = require('../models/timeentry');
-const badge = require('../models/badge');
-const myTeam = require('./helperModels/myTeam');
-const dashboardHelper = require('./dashboardhelper')();
-const reportHelper = require('./reporthelper')();
+const mongoose = require("mongoose");
```

Comment:
As @rauleffting said, please limit the changes you make with **Prettier**, because this PR aims to fix and improve a functionality not change stylistic changes to the code. This makes it hard to find the code you wrote that implements the new features, since the prettier package made a lot of automatic changes to the code.